tests/integration/pilot/vm/README.md

```diff
@@ -0,0 +1,58 @@
+# Writing Tests for VMs
+
+Support for including VMs in the integration tests has been added to the framework.
+This document describes the integration and extension mechanisms to exercise VM related code.
+The primary goals are to test VM-related code & ensure it works on a range of supported OS types and versions.
+
+**Note: We currently use mock/simulated VMs for testing purposes. In the future, the testing might switch
+to utilize actual compute instances from different providers.**
+
+## Deploy an Echo Instance as a VM
+
+To deploy an echo instance as a VM just set `DeployAsVm` to be true in `echo.Config`
```

Comment:
Restructure this section with step-by-step description. So users will first know what's the purpose of this section, and what are the steps to do it.